src/router/router.tsx

```diff
@@ -7,8 +7,61 @@ const router = createBrowserRouter([
     element: <HomePage />,
   },
   {
-    path: '/how-to-play',
-    element: <HomePage />,
+    path: 'how-to-play',
```

Comment:
You'll need an element here. Otherwise if the user types `/how-to-play` in the URL, they will get a blank page. You can add `<Outlet />` for now

Reply:
One other option: use the `<Navigate />` component to push them automatically to the overview page.

https://reactrouter.com/en/main/components/navigate